src/utils/log.ts

```diff
@@ -1,57 +1,64 @@
-import { stderr as chalkStderr } from "chalk";
-import { CliKintoneError } from "./error";
+import winston from "winston";
 
-const currentISOString = () => new Date().toISOString();
+const { combine, timestamp, printf, splat } = winston.format;
 
-export type Logger = {
-  debug: (message: any) => void;
-  info: (message: any) => void;
-  warn: (message: any) => void;
-  error: (message: any) => void;
-  fatal: (message: any) => void;
-};
+export type LogLevel = "debug" | "info" | "warn" | "error" | "fatal";
```

Comment:
Please make it clear whether it is log event level or log config level by its name.

Reply:
MUST: The "none" level is missing from the log config level.

Reply:
FYI: You can use `as const` to create a constant array and linked type.

```typescript
export const logEventLevels = [
  "debug",
  "info",
  "warn",
  "error",
  "fatal",
] as const;
type LogEventLevel = (typeof logEventLevels)[number];

export const logConfigLevels = [
  "debug",
  "info",
  "warn",
  "error",
  "fatal",
  "none",
] as const;
type LogConfigLevel = (typeof logConfigLevels)[number];
```